js/main.js

```diff
@@ -1244,20 +1419,24 @@ function displayLoaded() {
 }
 
 function init() {
-    createAllRows(jobCategories, 'jobTable');
+    createNestedRows(moduleCategories, 'jobTable');
     createAllRows(skillCategories, 'skillTable');
     createAllRows(itemCategories, 'itemTable');
 
     cleanUpDom();
 
     initSidebar();
 
-    createData(gameData.taskData, jobBaseData);
+    //direct ref assignment - taskData could be replaced
+    for (let entityData of Object.values(moduleOperations)) {
+        entityData.id = 'row ' + entityData.name;
+        gameData.taskData[entityData.name] = entityData
+    }

```

Comment:
Warum passiert das hier so "nackt" mitten in init()?

Reply:
🤷 Wenn man saves überarbeitet könnte die Taskdata auch weg.